code/modules/cm_aliens/structures/trap.dm

```diff
@@ -181,171 +178,171 @@
 			var/spray_type = get_spray_type(trap_type)
 
 			new spray_type(loc, cause_data, hivenumber)
-			for(var/turf/T in range(1,loc))
-				var/obj/effect/xenomorph/spray/SP = new spray_type(T, cause_data, hivenumber)
-				for(var/mob/living/carbon/H in T)
-					if(H.ally_of_hivenumber(hivenumber))
+			for(var/turf/Turf in range(1,loc))
+				var/obj/effect/xenomorph/spray/Acidsplash = new spray_type(Turf, cause_data, hivenumber)
+				for(var/mob/living/carbon/Human in Turf)
+					if(Human.ally_of_hivenumber(hivenumber))
 						continue
-					SP.apply_spray(H)
+					Acidsplash.apply_spray(Human)
```

Comment:
we use snake case, acid_splash, do this for everything instead of just adding capitals like Turf